src/test/scala/nl.knaw.dans.easy.deposit/servlets/HappyRoutesSpec.scala

```diff
@@ -169,7 +169,7 @@ class HappyRoutesSpec extends TestSupportFixture with ServletFixture with Scalat
       headers = Seq(("Authorization", fooBarBasicAuthHeader))
     ) {
       status shouldBe BAD_REQUEST_400
-      body shouldBe """Bad Request. invalid DatasetMetadata: class java.lang.Exception don't recognize {"title":"blabla"}"""
+      body shouldBe """Bad Request. invalid DatasetMetadata: java.lang.Exception: don't recognize {"title":"blabla"}"""
```

Comment:
is this a message that goes to the user/client? I would much rather not have exception class names like `java.lang.Exception` in there. Just a **very clear** error message.